.eslintrc.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     // React
     'react/jsx-filename-extension': ['error', { extensions: ['.js', '.mdx'] }],
     'react/destructuring-assignment': 'warn',
-    'react/jsx-props-no-spreading': 'warn',
+    'react/jsx-props-no-spreading': 'off',
```

Comment:
We should re-enable that :)

Reply:
How is it that we warn about prop spreading ? What if we need to spread props into a deep child component ?

Reply:
We might want to disable props spreading warning when it's justified yes. @chambo-e ?

Reply:
Yes, the end goal being to have explicit props on every components.

There might be some case where we'll still need to spread props (component pass through, hoc, ...) but we'll manage case by case :)